### PR TITLE
Update energy consumption page

### PR DESCRIPTION
**Pull request by jmcook1186**
## Description

Responding to issue #10109 .
Adds data from Cambridge source, removes estimate with less robust citation, and adds some detail to the text that clarifies uncertainty in energy consumption estimates. 

## Related Issue

#10109

### CODE REVIEW

File: src/content/energy-consumption/index.md [hunk 1]
```diff
@@ -6,36 +6,36 @@ lang: en
...
+| Gold mining           |                 131                 |          50,000x           | [source](https://ccaf.io/cbnsi/cbeci/comparisons)                                                                                                                                 |
+| Bitcoin               |                 131                 |          50,000x           | [source](https://ccaf.io/cbnsi/cbeci/comparisons)                                                                                                                                 |
+| PoW Ethereum          |                 78                  |          30,000x           | [source](https://digiconomist.net/ethereum-energy-consumption)                                                                                                                    |
+| Youtube (direct only) |                 12                  |           4600x            | [source](https://www.gstatic.com/gumdrop/sustainability/google-2020-environmental-report.pdf)                                                                                     |
```
Comment (minimalsm) [inline src/content/energy-consumption/index.md:25]: Did you mean to keep YouTube here? @jmcook1186

File: src/content/energy-consumption/index.md [hunk 2]
```diff
@@ -6,36 +6,36 @@ lang: en
...
+It is complicated to get accurate estimates for energy consumption, especially when what is being measured has a complex supply chain or deployment details that influence its efficiency. For example, the energy consumption for platforms such as Netflix and Youtube vary depending upon whether they only include the energy used to run the hardware infrastructure required to store and deliver content (_direct expenditure_) or whether they include the expenditure required to - for example - produce content, run corporate offices, advertise, etc (_indirect expenditure_). Indirect usage could also include the energy required to consume content on end-user devices such as TVs, computers and mobiles, which in turn depends on which devices are used - there is some discussion this on [Carbon Brief](https://www.carbonbrief.org/factcheck-what-is-the-carbon-footprint-of-streaming-video-on-netflix). In the table above, the value reported for Netflix is their self-reported total (_direct_ + _indirect_). Youtube estimates their own _direct_ energy expenditure at around [12 TWh/yr](https://www.gstatic.com/gumdrop/sustainability/google-2020-environmental-report.pdf).
```
Comment (minimalsm) [inline src/content/energy-consumption/index.md:32]: Tried to simplify the wording a little here. LMK what you think.

```suggestion
It is complicated to get accurate estimates for energy consumption, especially when what is being measured has a complex supply chain or deployment details that influence its efficiency. Consider Netflix and YouTube as examples. The energy they use is divided into two categories: direct and indirect. Direct energy is necessary to maintain their systems and deliver content to users. Indirect energy encompasses aspects such as producing shows, operating the company, advertising, and even the energy consumed while watching on devices like TVs or phones. 

You can learn more about these differences at [Carbon Brief](https://www.carbonbrief.org/factcheck-what-is-the-carbon-footprint-of-streaming-video-on-netflix). In the table above, the number for Netflix includes both direct and indirect energy. YouTube, on the other hand, only provides us a number for their direct energy use, which is around [12 TWh/yr](https://www.gstatic.com/gumdrop/sustainability/google-2020-environmental-report.pdf).
```

Comment (minimalsm) [reply src/content/energy-consumption/index.md:32]: let me come back to this tomorrow, i think I've lost a bit of the nuance. Netflix's number is direct + indirect but does not include end users..

File: src/content/energy-consumption/index.md [hunk 3]
```diff
@@ -58,17 +58,18 @@ While Ethereum's energy consumption is very low, there is also a substantial, gr
 Web3 native public goods funding platforms such as [Gitcoin](https://gitcoin.co) run climate rounds to stimulate environmentally conscious building on Ethereum's application layer. Through the development of these initiatives (and others, e.g. [DeSci](/desci/)), Ethereum is becoming an environmentally and socially net-positive technology.
 
 <InfoBanner emoji=":evergreen_tree:">
-  If you think this page can be made more accurate, please raise an issue or PR. The stats on this page are estimates based on publicly available data - they do not represent an official statement or promise from the ethereum.org team, or the Ethereum Foundation. 
+  If you think this page can be made more accurate, please raise an issue or PR. The stats on this page are estimates based on publicly available data - they do not represent an official statement or promise from the ethereum.org team, or the Ethereum Foundation.
 </InfoBanner>
 
 ## Further reading {#further-reading}
 
+- [Cambridge Blockchain Network Sustainability Index ](https://ccaf.io/cbnsi/ethereum)
```
Comment (minimalsm) [inline src/content/energy-consumption/index.md:66]:
```suggestion
- [Cambridge Blockchain Network Sustainability Index](https://ccaf.io/cbnsi/ethereum)
```

File: src/intl/en/page-about.json [hunk 4]
```diff
@@ -24,14 +24,13 @@
...
-  "energy-consumption-chart-gold-mining-galaxy-label": "Gold mining (Galaxy Digital)",
   "energy-consumption-chart-global-data-centers-label": "Global data centers",
-  "energy-consumption-chart-gold-mining-cbeci-label": "Gold mining (CBECI)",
+  "energy-consumption-gold-mining-cbeci-label": "Gold mining",
   "energy-consumption-chart-btc-pow-label": "BTC PoW",
   "energy-consumption-chart-netflix-label": "Netflix",
   "energy-consumption-chart-eth-pow-label": "ETH PoW",
   "energy-consumption-chart-gaming-us-label": "Gaming in the US",
   "energy-consumption-chart-paypal-label": "PayPal",
+  "energy-consumption-chart-airbnb-label": "AirBnB",
```
Comment (minimalsm) [inline src/intl/en/page-about.json:34]: Aside: any point in translating these?

cc: @lukassim

File: src/content/energy-consumption/index.md [hunk 5]
```diff
@@ -6,36 +6,36 @@ lang: en
 
 # Ethereum's energy expenditure {#proof-of-stake-energy}
 
-Ethereum is a green blockchain. It uses a [proof-of-stake](/developers/docs/consensus-mechanisms/pos) consensus mechanism, using ETH instead of [energy to secure the network](/developers/docs/consensus-mechanisms/pow). Ethereum's proof-of-stake mechanism only uses [~0.0026 TWh/yr](https://carbon-ratings.com/eth-report-2022) across the entire global network.
+Ethereum is a green blockchain. Ethereum's [proof-of-stake](/developers/docs/consensus-mechanisms/pos) consensus mechanism uses ETH instead of [energy to secure the network](/developers/docs/consensus-mechanisms/pow). Ethereum's expends approximately [~0.0026 TWh/yr](https://carbon-ratings.com/eth-report-2022) across the entire global network.
```
Comment (minimalsm) [inline src/content/energy-consumption/index.md:9]:
```suggestion
Ethereum is a green blockchain. Ethereum's [proof-of-stake](/developers/docs/consensus-mechanisms/pos) consensus mechanism uses ETH instead of [energy to secure the network](/developers/docs/consensus-mechanisms/pow). Ethereum's energy consumption is approximately [~0.0026 TWh/yr](https://carbon-ratings.com/eth-report-2022) across the entire global network.
```

File: src/content/energy-consumption/index.md [hunk 6]
```diff
@@ -6,36 +6,36 @@ lang: en
...
+The energy consumption estimate for Ethereum comes from a [CCRI (Crypto Carbon Ratings Institute)](https://carbon-ratings.com) study. They generated bottom-up estimates of the electricity consumption and the carbon footprint of the Ethereum network ([see the report](https://carbon-ratings.com/eth-report-2022)). They measured the electricity consumption of different nodes with various hardware and client software configurations. The estimated **2,601 MWh** (0.0026 TWh) for the network’s annual electricity consumption corresponds to yearly carbon emissions of **870 tonnes CO2e** applying regional-specific carbon intensity factors. This value changes as nodes enter and leave the network - you can keep track using a rolling 7-day average estimate by [Cambridge Blockchain network Sustainability Index](https://ccaf.io/cbnsi/ethereum) (note that they use a slightly different method for their estimates - details available on their site).
+
+To put Ethereum's energy consumption in context, we can compare annualized estimates for some other industries - this allows us to better understand whether the estimate for Ethereum is a lot or a little.
```
Comment (minimalsm) [inline src/content/energy-consumption/index.md:13]:
```suggestion
To contextualize Ethereum's energy consumption, we can compare annualized estimates for some other industries. This helps us better understand whether the estimate for Ethereum is high or low.
```

File: src/content/energy-consumption/index.md [hunk 7]
```diff
@@ -6,36 +6,36 @@ lang: en
...
+The energy consumption estimate for Ethereum comes from a [CCRI (Crypto Carbon Ratings Institute)](https://carbon-ratings.com) study. They generated bottom-up estimates of the electricity consumption and the carbon footprint of the Ethereum network ([see the report](https://carbon-ratings.com/eth-report-2022)). They measured the electricity consumption of different nodes with various hardware and client software configurations. The estimated **2,601 MWh** (0.0026 TWh) for the network’s annual electricity consumption corresponds to yearly carbon emissions of **870 tonnes CO2e** applying regional-specific carbon intensity factors. This value changes as nodes enter and leave the network - you can keep track using a rolling 7-day average estimate by [Cambridge Blockchain network Sustainability Index](https://ccaf.io/cbnsi/ethereum) (note that they use a slightly different method for their estimates - details available on their site).
```
Comment (minimalsm) [inline src/content/energy-consumption/index.md:11]:
```suggestion
The energy consumption estimate for Ethereum comes from a [CCRI (Crypto Carbon Ratings Institute)](https://carbon-ratings.com) study. They generated bottom-up estimates of the electricity consumption and the carbon footprint of the Ethereum network ([see the report](https://carbon-ratings.com/eth-report-2022)). They measured the electricity consumption of different nodes with various hardware and client software configurations. The estimated **2,601 MWh** (0.0026 TWh) for the network’s annual electricity consumption corresponds to yearly carbon emissions of **870 tonnes CO2e** applying regional-specific carbon intensity factors. This value changes as nodes enter and leave the network - you can keep track using a rolling 7-day average estimate by the [Cambridge Blockchain network Sustainability Index](https://ccaf.io/cbnsi/ethereum) (note that they use a slightly different method for their estimates - details available on their site).
```

File: src/content/energy-consumption/index.md [hunk 8]
```diff
@@ -6,36 +6,36 @@ lang: en
...
+
+To put Ethereum's energy consumption in context, we can compare annualized estimates for some other industries - this allows us to better understand whether the estimate for Ethereum is a lot or a little.
 
 <EnergyConsumptionChart />
 
-The chart above shows the estimated annual energy consumption in TWh/yr for various industries (retrieved in June 2022). Note that the estimates presented in the plot are from publicly available sources that have been linked to in the table below. CEBCI refers to the Cambridge Bitcoin Electricity Consumption index. The values are illustrative and do not represent an official estimate, promise, or forecast.
+The chart above shows the estimated annual energy consumption in TWh/yr for Ethereum alongside some other industries. The estimates presented are from publicly available sources, accessed in May 2023, that are linked to in the table below:
```
Comment (minimalsm) [inline src/content/energy-consumption/index.md:17]:
```suggestion
The chart above displays the estimated yearly energy consumption in TWh/yr for Ethereum, compared to several other industries. The estimates provided are sourced from publicly available information, accessed in May 2023, with links to the sources available in the table below:
```

File: src/content/energy-consumption/index.md [hunk 9]
```diff
@@ -45,11 +45,11 @@ For example, on Ethereum, the transaction throughput is not only that of the bas
...
 
-CCRI examined the impact of Ethereum’s transition from proof-of-work to proof-of-stake. The annualized electricity consumption was reduced by more than **99.988%**. Likewise, Ethereum’s carbon footprint was decreased by approximately **99.992%** (from 11,016,000 to 870 tonnes CO2e). Depicted metaphorically, this corresponds to a reduction in emissions from the height of the Eiffel Tower to a small plastic toy figure, as shown in the figure above. Therefore, the environmental cost of securing the network is drastically reduced. Simultaneously, the security of the network is thought to have increased.
+CCRI estimate that The Merge reduced Ethereum's annualized electricity consumption by more than **99.988%**. Likewise, Ethereum’s carbon footprint was decreased by approximately **99.992%** (from 11,016,000 to 870 tonnes CO2e). Depicted metaphorically, this corresponds to a reduction in emissions from the height of the Eiffel Tower to a small plastic toy figure, as shown in the figure above. Therefore, the environmental cost of securing the network is drastically reduced. Simultaneously, the security of the network is thought to have increased.
```
Comment (minimalsm) [inline src/content/energy-consumption/index.md:52]:
```suggestion
CCRI estimate that The Merge reduced Ethereum's annualized electricity consumption by more than **99.988%**. Likewise, Ethereum’s carbon footprint was decreased by approximately **99.992%** (from 11,016,000 to 870 tonnes CO2e). To put this in perspective, the reduction in emissions is like going from the height of the Eiffel Tower to a small plastic toy figure, as illustrated in the figure above. As a result, the environmental cost of securing the network is drastically reduced. At the same time, the network's security is believed to have improved.
```

File: src/content/energy-consumption/index.md [hunk 10]
```diff
@@ -45,11 +45,11 @@ For example, on Ethereum, the transaction throughput is not only that of the bas
...
+From the very beginning, Ethereum planned to implement a proof-of-stake based consensus mechanism, but doing so without sacrificing security and decentralization took years of focused research and development. Therefore, a proof-of-work mechanism was used to get the network started. Proof-of-work requires miners to use their computing hardware to calculate a value, expending energy in the process.
 
 ![Energy consumption comparison of pre- and post-Merge Ethereum. Displayed is on the left the Eiffel tower with 330 meters height and on the right a plastic toy figure with 4 cm height within a magnifying glass.](energy_consumption_pre_post_merge.png)
```
Comment (minimalsm) [inline src/content/energy-consumption/index.md:50]: Slight wording improvement + Magnifying glass isn't really relevant to describe this image.

```suggestion
![An illustration comparing Ethereum's energy consumption pre- and post-Merge, using the Eiffel Tower (330 meters tall) on the left to symbolize the high energy consumption before The Merge, and a small 4 cm tall Lego figure on the right to represent the dramatic reduction in energy usage after The Merge](energy_consumption_pre_post_merge.png)
```

File: src/content/energy-consumption/index.md (hunk 2 above)
Comment (minimalsm) [inline src/content/energy-consumption/index.md:32]:
```suggestion
It is complicated to get accurate estimates for energy consumption, especially when what is being measured has a complex supply chain or deployment details that influence its efficiency. Consider Netflix and YouTube as examples. The energy they use can be divided into two types: direct and indirect. Direct energy is what's needed to keep their systems running and deliver content to you. Indirect energy includes things like making the shows, keeping the company going, advertising, and even the energy you use to watch on your TV or phone.

However, it's essential to note that the indirect energy consumption reported by platforms usually won't include the substantial energy usage associated with millions of users watching content on their devices. This discrepancy can lead to underestimating the true energy consumption for these platforms. You can learn more about these differences at [Carbon Brief](https://www.carbonbrief.org/factcheck-what-is-the-carbon-footprint-of-streaming-video-on-netflix). In the table above, the value reported for Netflix encompasses both direct and indirect energy while YouTube only offers information on their direct energy consumption, which amounts to around [12 TWh/yr](https://www.gstatic.com/gumdrop/sustainability/google-2020-environmental-report.pdf).
```

File: src/intl/en/page-about.json (hunk 4 above)
Comment (lukassim) [reply src/intl/en/page-about.json:34]: Probably not, 'Gaming in the US' is the only one that would get translated in the vast majority of languages. There's also some languages that write names like PayPal & AirBnB in their native script, but in most languages names are still written using Latin characters

**Comment by kuzdogan** [comment]
Should we maybe make the warning on the estimation difficulty a little more visible? Like an [Admonition](https://docusaurus.io/docs/markdown-features/admonitions) as called in Docusaurus. 

image

A proper place IMO would be right before the energy consumption table but after the chart as the chart gives a good quick look but it could be nice to underline this is a (difficult) estimation before we present numbers and sources.

+ a small thing: I'd suggest making "ETH PoS" text on the chart bold or using another way to highlight it, as it is the main element we want to draw attention to and the YouTube's bright red is drawing way too much attention. 
image

File: src/content/energy-consumption/index.md (hunk 2 above)
Comment (jmcook1186) [reply src/content/energy-consumption/index.md:32]:
```suggestion
It is complicated to get accurate estimates for energy consumption, especially when what is being measured has a complex supply chain or deployment details that influence its efficiency. consider Netflix or Youtube as examples. Estimates of their energy consumption vary depending upon  whether they only include the energy used to maintain their systems and deliver content to users (_direct expenditure_) or whether they include the expenditure required to produce content, run corporate offices, advertise, etc (_indirect expenditure_). Indirect usage could also include the energy required to consume content on end-user devices such as TVs, computers and mobiles, which in turn depends on which devices are used.

There is some discussion of this issue on [Carbon Brief](https://www.carbonbrief.org/factcheck-what-is-the-carbon-footprint-of-streaming-video-on-netflix). In the table above, the value reported for Netflix includes their self-reported _direct_ and _indirect_ usage. Youtube only provides an estimate of their own _direct_ energy expenditure, which is around [12 TWh/yr](https://www.gstatic.com/gumdrop/sustainability/google-2020-environmental-report.pdf).
```

**Review by corwintines — APPROVED** [review]

File: src/intl/en/page-about.json (hunk 4 above)
Comment (nhsz) [reply src/intl/en/page-about.json]: should we should remove these labels then?

- "energy-consumption-chart-btc-pow-label"
- "energy-consumption-chart-netflix-label"
- "energy-consumption-chart-eth-pow-label"
- "energy-consumption-chart-paypal-label"
- "energy-consumption-chart-airbnb-label"

cc @lukassim 

I'll open a PR if it's ok
